example/autoencoder/convolutional_autoencoder.ipynb

```diff
@@ -0,0 +1,578 @@
+      "text/plain": [
+       "<Figure size 1440x288 with 20 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(20,4))\n",
+    "for i in range(10):\n",
+    "    idx = random.randint(0, len(test_dataset))\n",
+    "    img, _ = test_dataset[idx]\n",
+    "    x, _ = test_dataset_t[idx]\n",
+    "\n",
+    "    data = x.as_in_context(ctx).expand_dims(axis=0)\n",
+    "    output = net(data)\n",
+    "
```

Comment:
Kind of looks like we're interpolating along two axis here since 5x5 grid. Can we just have one line?
Maybe add some more images, have 10 smaller images in one line, like earlier on in notebook.

Reply:
I think I disagree with that but I have updated as you suggested :+1: